core/src/main/java/com/scalar/db/config/DatabaseConfig.java

```diff
@@ -88,7 +88,7 @@ protected void load() {
     metadataCacheExpirationTimeSecs =
         getLong(getProperties(), METADATA_CACHE_EXPIRATION_TIME_SECS, -1);
     activeTransactionManagementExpirationTimeMillis =
-        getLong(getProperties(), ACTIVE_TRANSACTION_MANAGEMENT_EXPIRATION_TIME_MILLIS, 0);
+        getLong(getProperties(), ACTIVE_TRANSACTION_MANAGEMENT_EXPIRATION_TIME_MILLIS, -1);
```

Comment:
Changed the default value of the property `scalar.db.active_transaction_management.expiration_time_millis` for consistency. However, it's not backward incompatible change because values less than or equal to `0` are treated the same as before.